tests/main.test.ts

```diff
@@ -382,6 +372,24 @@ async function setupTests() {
     owner: "ubiquity",
     repo: "test-repo",
   });
+
+  db.issue.create({
+    ...issueTemplate,
+    id: 7,
+    assignee: {
```

Comment:
You have created two new issues which are currently assigned to users who only exist here, I can't find mention of them anywhere else. If that is the case then how can they have reached their max assigned tasks?

If the issue is already assigned when someone calls `/start` on it (and you have loaded these new issues into your tests) then shouldn't it be handled with `This task is already assigned. Please choose another task.`?

Reply:
I made some changes regarding these this morning please can you check them out?

Reply:
As far as I can tell you have removed the new issues you created and have created two new users in the DB that have no assigned issues, so they'll pass your validation checks.

If there is a config limit of 10 for admins, then there would need to be at least ten issues right? Or we could lower the limit we test with and then have logs that we use `expect()` on and confirm the input/output/outcome specifically so there is no doubt

Reply:
I've made changes to match your recommendations. I've added issues to the users I created and tested the test fail when they should and pass when the should.